src/NuGet.Services.EndToEnd/Support/Clients/V2V3SearchClient.cs

```diff
@@ -234,7 +208,7 @@ public async Task<IReadOnlyList<SearchServiceProperties>> GetSearchServicesAsync
             return searchServices;
         }
 
-        private async Task<SearchServiceProperties> GetSearchServiceFromAzureAsync(
```

Comment:
This will be a no-async method. Maybe it's better to name it without Async like "GetSearchServiceProperties"?